test_component_fast/test_component_fast.vcxproj

```diff
@@ -46,6 +46,7 @@
     <PlatformToolset>v140</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)' == '15.0'">v141</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '17.0'">v143</PlatformToolset>
```

Comment:
It might make sense to just set it to v142 always as that is what testcomponent and so on does, but this is fine too.